userspace/ksud/src/defs.rs

```diff
@@ -12,9 +12,11 @@ pub const PROFILE_TEMPLATE_DIR: &str = concatcp!(PROFILE_DIR, "templates/");
 pub const KSURC_PATH: &str = concatcp!(WORKING_DIR, ".ksurc");
 pub const KSU_OVERLAY_SOURCE: &str = "KSU";
 pub const DAEMON_PATH: &str = concatcp!(ADB_DIR, "ksud");
+pub const MAGISKBOOT_PATH: &str = concatcp!(ADB_DIR, "magisk");
```

Comment:
We can put `magiskboot` directly in `/data/adb/ksu/bin/`. There is no need to create a symlink, as `/data/adb/ksu/bin` is already added to the `$PATH` for su. You can use `magiskboot` without specifying its full path.